alcs-api/src/application/application-subtask/application-subtask.entity.ts

```diff
@@ -0,0 +1,33 @@
+import { Column, CreateDateColumn, Entity, ManyToOne } from 'typeorm';
+import { Base } from '../../common/entities/base.entity';
+import { User } from '../../user/user.entity';
+import { Application } from '../application.entity';
+import { ApplicationSubtaskType } from './application-subtask-type.entity';
+
+@Entity()
+export class ApplicationSubtask extends Base {
+  constructor(data?: Partial<ApplicationSubtask>) {
+    super();
+    if (data) {
+      Object.assign(this, data);
+    }
+  }
+
+  @CreateDateColumn({ type: 'timestamptz' })
+  createdAt: Date;
+
+  @Column({ nullable: true })
```

Comment:
Do we not specify timestamptz for this attribute as well?

Reply:
It should be, good catch